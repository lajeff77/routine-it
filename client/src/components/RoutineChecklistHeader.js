import React from "react";
import {Box, Typography} from "@mui/material";

const RoutineChecklistHeader = (props) => {
    return(
        <Box>
            <Typography variant="h1" sx={{textAlign: 'center', margin: '30px'}}>{props.name}</Typography>
        </Box>
    )
}

export default RoutineChecklistHeader;