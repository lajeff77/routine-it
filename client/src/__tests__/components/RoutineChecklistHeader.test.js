import { render } from "@testing-library/react";
import {expect, test} from '@jest/globals';
import React from "react";
import RoutineChecklistHeader from "../../components/RoutineCheckListPage/RoutineChecklistHeader";

const testName = "Test Routine";
describe(RoutineChecklistHeader.name, ()=>{
    test('RoutineChecklistHeader matches snapshot', () => {
        const {container} = render(<RoutineChecklistHeader name={testName}/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});