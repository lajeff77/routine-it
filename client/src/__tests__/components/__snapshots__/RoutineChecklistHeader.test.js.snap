// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoutineChecklistHeader RoutineChecklistHeader matches snapshot 1`] = `
<div
  class="MuiBox-root css-0"
>
  <h1
    class="MuiTypography-root MuiTypography-h1 css-3ns5jg-MuiTypography-root"
  >
    Test Routine
  </h1>
</div>
`;
